src/common/dcsvg.cpp

```diff
@@ -46,35 +46,40 @@ namespace
 // C locale (i.e. always using "." for the decimal separator) and with the
 // fixed precision (which is 2 for some unknown reason but this is what it was
 // in this code originally).
-inline wxString NumStr(double f)
+inline wxString NumStr(double const& f)
 {
-    return wxString::FromCDouble(f, 2);
+    if ( f == 0 )
```

Comment:
Why does this need to be special-cased? Is it an optimization?

Reply:
To prevent strings with `-0.00`. I added a comment.